less/home-page.less

```diff
@@ -1,61 +1,161 @@
-.home {
-
-  .content-section {
-    margin: 30px 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-
-   
+.home
+  
+  .logo{
+    @media only screen and (max-width:500px) {
+      align-self: center;
+      padding-left: 2rem;
+      font-size: 2rem;
+    }
+  }
+  
+  
+  body {
```

Comment:
Why are you targeting the body here?  You're also putting a border at the bottom of the page